extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/user/AuthenticatedUser.java

```diff
@@ -53,6 +61,72 @@ public void init(Credentials credentials) {
         setIdentifier(credentials.getUsername());
     }
 
+    /**
+     * Get a map of attributes associated with this AuthenticatedUser.
+     *
+     * @return
+     *     The Map of arbitrary attributes associated with this
+     *     AuthenticatedUser object.
+     */
```

Comment:
Since this is overriding another method, I think it doesn't need the Javadocs, no?